include/UtilFunctionTesters/UtilFunctionTester.h

```diff
@@ -5,6 +5,9 @@
 #include "Debug.h"
 #include "Utils.h"
 
+// a purely abstract class to be extended in order to represent
```

Comment:
Is "purely abstract" different than "abstract" in this context? I think not, and "pure" has a technical meaning (deterministic and side-effect-free) when discussing programs, so avoid using the word if it's not needed.